query/graphql/src/main/java/org/codice/ddf/admin/graphql/servlet/ExtendedOsgiGraphQLServlet.java

```diff
@@ -141,8 +173,12 @@ protected void doPost(HttpServletRequest originalRequest, HttpServletResponse or
                             "[" + String.join(",", responses) + "]" :
                             responses.get(0));
         } catch (Throwable t) {
-            originalResponse.setStatus(500);
-            log.error("Error executing GraphQL request!", t);
+            if(t instanceof RuntimeException) {
```

Comment:
✏️ Instead of doing an `instanceof` check, have multiple `catch` blocks:
```java
// ..
} catch (RuntimeException re) {
    originalResponse.setStatus(500);
} catch (Throwable t) {
    originalResponse.setStatus(400);
}
```